python/Ganga/Core/GangaRepository/GangaRepository.py

```diff
@@ -205,7 +205,7 @@ def _make_empty_object_(self, this_id, category, classname):
         #obj.__init__()
         obj.setNodeData({})
         obj.setNodeAttribute('id', this_id)
-
+        obj._setFlushed()
```

Comment:
Ah, so this function naming is confusing. It doesn't create an `EmptyGangaObject` but rather simple an object of the correct type with its data emptied out. What is the purpose of the `_make_empty_object_` function in this case?